release/run_tests_release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+function main {
+	find . -name "test_*.sh" -type f ! -name "test_bom.sh" -exec ./{} \;
```

Comment:
@lucasmiranda0, can you please remove `-name test_bom.sh`?

Also, can you please make sure that `test_bom.sh` will be skipped if there is no `liferay-portal-ee` repo available?